[2023-12-20T12:01:27.156+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v4.second_id scheduled__2022-10-22T00:00:00+00:00 [queued]>
[2023-12-20T12:01:27.237+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v4.second_id scheduled__2022-10-22T00:00:00+00:00 [queued]>
[2023-12-20T12:01:27.237+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 6
[2023-12-20T12:01:27.439+0000] {taskinstance.py:2192} INFO - Executing <Task(BashOperator): second_id> on 2022-10-22 00:00:00+00:00
[2023-12-20T12:01:27.461+0000] {standard_task_runner.py:60} INFO - Started process 10796 to run task
[2023-12-20T12:01:27.477+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v4', 'second_id', 'scheduled__2022-10-22T00:00:00+00:00', '--job-id', '2928', '--raw', '--subdir', 'DAGS_FOLDER/first_dat.py', '--cfg-path', '/tmp/tmpf54pnfbb']
[2023-12-20T12:01:27.482+0000] {standard_task_runner.py:88} INFO - Job 2928: Subtask second_id
[2023-12-20T12:01:27.714+0000] {task_command.py:423} INFO - Running <TaskInstance: our_first_dag_v4.second_id scheduled__2022-10-22T00:00:00+00:00 [running]> on host 29b3723064ac
[2023-12-20T12:01:28.707+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='abrham' AIRFLOW_CTX_DAG_ID='our_first_dag_v4' AIRFLOW_CTX_TASK_ID='second_id' AIRFLOW_CTX_EXECUTION_DATE='2022-10-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-10-22T00:00:00+00:00'
[2023-12-20T12:01:28.709+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-12-20T12:01:28.710+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and will be runing after task 1']
[2023-12-20T12:01:28.823+0000] {subprocess.py:86} INFO - Output:
[2023-12-20T12:01:28.862+0000] {subprocess.py:93} INFO - hey, I am task2 and will be runing after task 1
[2023-12-20T12:01:28.891+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-12-20T12:01:29.127+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v4, task_id=second_id, execution_date=20221022T000000, start_date=20231220T120127, end_date=20231220T120129
[2023-12-20T12:01:29.693+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2023-12-20T12:01:29.783+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
